src/components/Icon/iconDefinitions.js

```diff
@@ -110,6 +110,16 @@ export default {
       </g>
     ),
   },
+  speechBubble: {
+    title: 'Speech bubble',
+    viewBox: '0 0 29 25',
+    svgClassName: 'speechBubble',
+    body: (
+      <g transform="translate(-1402.000000, -643.000000)">
```

Comment:
I'm disappointed that you didn't go and subtract `1402` from all the x value and `643` from all the y values so that you could remove this.

Also, serious note the `id="shape"` can go.

Reply:
lol!